processing/src/main/java/io/druid/query/extraction/ExtractionFn.java

```diff
@@ -71,13 +74,9 @@
    */
   public String apply(Object value);
 
-  public String apply(String value);
```

Comment:
but this is backward incompatible ? it is totally a new API, can we have a proposal and maybe come with something compatible ?